test/lib/runtime/flow_test.exs

```diff
@@ -1304,7 +1304,7 @@ defmodule Ask.FlowTest do
 
   defp reply_ivr(flow, reply), do: test_reply(flow, "ivr", reply)
 
-  defp test_reply(flow, mode, reply, old_disposition \\ "any_disposition"), do:
+  defp test_reply(flow, mode, reply, old_disposition \\ :contacted), do:
```

Comment:
Because we can't use whatever disposition name anymore, it must be a valid value, otherwise we get a runtime error that the disposition doesn't exist.